_drafts/dealing-with-large-symbol-files.md

```diff
@@ -0,0 +1,592 @@
+---
+title: Dealing with Large Symbol Files
+description: Some tools and techniques for reducing debug symbol file sizes
+author: noah
+image: # TODO
+tags: [gdb, fw-code-size]
+---
+
+<!-- excerpt start -->
+
+Large applications can produce _very_ large symbol files when debug information
+is enabled (especially at the higher, more verbose levels of debug info!). This
+article describes some techniques for reducing debug symbol file sizes!
+
+<!-- excerpt end -->
+
+{% include newsletter.html %}
+
+{% include toc.html %}
+
```

Comment:
add a note referencing https://interrupt.memfault.com/blog/gnu-binutils since we use a couple of those tools in this article